docs/cloud/integrations/golang.md

```diff
@@ -0,0 +1,142 @@
+---
+title:  Connect to Steampipe Cloud with Golang
+sidebar_label: Golang
+---
+# Connect to Steampipe Cloud from Golang
+
+Since your Steampipe Cloud workspace is a Postgres database, by using the `pq` driver you can connect and query your workspace database with Golang.
+
+The [Connect](/docs/cloud/integrations/overview) tab for your workspace provides the details you need to connect Steampipe Cloud with Golang.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/steampipe-cloud-connect-details.jpg" />
+</div>
+
+You'll need the Postgres driver for Go's database/sql package, which you can install using `go get github.com/lib/pq`.
+Then you specify the connection string, create a connection, run a query, and fetch results. In this example, we query the name, region and the versioning state of `aws_s3_bucket`.
+
+```go
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/lib/pq"
+)
+
+var db *sql.DB
+func main() {
+var err error
+
+connStr := "postgresql://rahulsrivastav14:f3**-****-**2c@rahulsrivastav14-rahulsworkspace.usea1.db.steampipe.io:9193/dea4px"
+db, err = sql.Open("postgres", connStr)
+if err != nil {
+panic(err)
+}
+if err = db.Ping(); err != nil {
+panic(err)
+}
+rows, err := db.Query(`select name, region, versioning_enabled from aws_s3_bucket`)
+defer rows.Close()
+
+var name string
+var region string
+var versioning_enabled string
+
+	for rows.Next() {
+	switch err := rows.Scan(&name, &region, &versioning_enabled); err {
+	case sql.ErrNoRows:
+		fmt.Println("No matching rows")
+	case nil:
+		fmt.Println(name, region, versioning_enabled, "\n")
+	default:
+		panic(err)
+		}
+	}
+}
```

Comment:
The formatting is off here. Go has a lovely approach: You run `gofmt -s -w .` and it decides how things should be.

Reply:
Thank you for sharing the command. I've formatted the code and pasted it to the doc.